src/config/backends/index.ts

```diff
@@ -5,13 +5,15 @@ import { ExtraUEnv } from './extra-uEnv';
 import { ConfigTxt } from './config-txt';
 import { ConfigFs } from './config-fs';
 import { Odmdata } from './odmdata';
+import { SplashImage } from './splash';
```

Comment:
I think changing the backend to splash-image makes it more clear that this backend only supports images in case splash could be interpreted as something else.